`@aws-sdk/querystring-builder` is not correctly marked as a dependency.

It is used in `src/dispatch/BeaconHttpHandler.ts`, which causes errors when using Plug-n-Play build tooling:

```
Module not found: Error: Can't resolve '@aws-sdk/querystring-builder' in '/home/runner/work/.../.../.yarn/cache/aws-rum-web-npm-1.12.0-d1cba678f0-d2387b42c7.zip/node_modules/aws-rum-web/dist/es/dispatch'
```

I believe this can be solved as simply as adding `"@aws-sdk/querystring-builder": "^3.0.0"` to `package.json`'s `dependencies` list.

--- a/package.json
+++ b/package.json
@@ -120,6 +120,7 @@
         "@aws-crypto/sha256-js": "^2.0.2",
         "@aws-sdk/fetch-http-handler": "^3.36.0",
         "@aws-sdk/protocol-http": "^3.36.0",
+        "@aws-sdk/querystring-builder": "^3.36.0",
         "@aws-sdk/signature-v4": "^3.36.0",
         "@aws-sdk/util-hex-encoding": "^3.36.0",
         "@babel/runtime": "^7.16.0",

